cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -5330,22 +5602,6 @@ mod tests {
                 .exactly_one_underline("r")
                 .build(),
         );
-        expect_some_error_matches(
-            src,
-            &errs,
-            &ExpectedErrorMessageBuilder::error("invalid variable: w")
-                .help("the valid Cedar variables are `principal`, `action`, `resource`, and `context`; did you mean to enclose `w` in quotes to make a string?")
-                .exactly_one_underline("w")
-                .build(),
-        );
-        expect_some_error_matches(
-            src,
-            &errs,
-            &ExpectedErrorMessageBuilder::error("invalid variable: u")
-                .help("the valid Cedar variables are `principal`, `action`, `resource`, and `context`; did you mean to enclose `u` in quotes to make a string?")
-                .exactly_one_underline("u")
-                .build(),
-        );
```

Comment:
can we leave these tests in?  (gated by `#[cfg(not(feature = "tolerant-ast"))]` if necessary)

Reply:
Will fix

Reply:
Ah sorry, looked closer at this. This whole test is guarded ` #[cfg(feature = "tolerant-ast")]` and uses the tolerant version of the parser which is why it has fewer error messages - these invalid expressions are correctly being translated into error nodes in the tolerant AST

There is another, separate test `show_policy1` which uses the normal parsing methods and it has these test cases

Reply:
Cool 👍🏻